NEWS.md

```diff
@@ -178,6 +178,8 @@ Library improvements
     * `charwidth(c)` and `strwidth(s)` now return up-to-date cross-platform
       results (via utf8proc) ([#10659]): Julia now likes pizza ([#3721]), but some terminals still don't.
 
+    * `is_valid_char(c)` now correctly handles Unicode "non-characters", which are valid Unicode codepoints.
```

Comment:
Cross reference the issue this fixes (or pull request) as you see in the above. There's a script somewhere (maybe in contrib? got an obvious name but I'm on my tablet and lazy) to run after which will generate the correct footnote.